core/src/main/java/org/frankframework/util/MessageUtils.java

```diff
@@ -328,4 +329,28 @@ public static long computeSize(Message message) {
 			return Message.MESSAGE_SIZE_UNKNOWN;
 		}
 	}
+
+	/**
+	 * Convert an object to a string. Does not close object when it is of type Message or MessageWrapper.
+	 */
+	@Deprecated
+	public static String asString(Object object) throws IOException {
```

Comment:
As discussed with Niels, I moved this method to the `MessageUtils` class for the cases that weren't that easily modified (see #7078).

Reply:
Conceptually this method has nothing to do with Message class so I think it should be in some other sort of utility class until we figured out how to do this better -- ConversionUtils or something like that perhaps?

@nielsm5 What do you think?

Reply:
I think that for now it's fine as long as we start to work on #7078 soon to hopefully get rid of this altogether.
Luckily there are only 5 implementations left.